README.md

```diff
@@ -1,96 +1,119 @@
 
-### Project Feedback + Evaluation
+### Improvements to be made
+**Walls:** I had tried to apply a for loop on Walls to make multiple horizontal obstacles. However, it seemed it did not work on multiple Walls beyond the first. Troubleshooting it didn't really work either. In the end, I opted to work with this restriction and include just one wall in each level set up.
 
-* __Project Workflow__: Did you complete the user stories, wireframes, task tracking, and/or ERDs, as specified above? Did you use source control as expected for the phase of the program you’re in (detailed above)?
+**Level setup:** Honestly the levels could be improved much more, but most of my project time was allocated to fine tuning the mechanics. The placement of the platforms and walls can be adjusted further to give the player a more rewarding gameplay experience, e.g. more required use of asymmetric movements between the balls.
 
-* __Technical Requirements__: Did you deliver a project that met all the technical requirements? Given what the class has covered so far, did you build something that was reasonably complex?
+**CSS:** Not the most enticing game to play at first glance. My h1 element has some whitespace which I unsuccessfully tried to remove. (but that's not the main point) I did try to make the display as simple as possible, summarize the instructions etc. Also, improve the 'juiciness' of the game such as bounciness of the platforms upon landing.
 
-* __Creativity__: Did you add a personal spin or creative element into your project submission? Did you deliver something of value to the end user (not just a login button and an index page)?
+**Smelly code:**
+I believe there are ways to be more efficient at coding, like reducing how many times an instance appears. DRY and KISS: I'm still learning!
 
-* __Code Quality__: Did you follow code style guidance and best practices covered in class, such as spacing, modularity, and semantic naming? Did you comment your code as your instructors have in class?
 
-* __Deployment__: Did you deploy your application to a public url using GitHub Pages?
+### References
+Not much here, unless online resources such as stack overflow, MDN, are to be counted, then I'd have a long list.
 
-* __Total__: Your instructors will give you a total score on your project between:
+I used free sound files via [freesound.org](http://freesound.org) which I linked the author's profiles to at the bottom of my game.
 
-    Score | Expectations
-    ----- | ------------
-    **0** | _Incomplete._
-    **1** | _Does not meet expectations._
-    **2** | _Meets expectations, good job!_
-    **3** | _Exceeds expectations, you wonderful creature, you!_
+I also used wikipedia for the gravity formula, but it seems the page no longer looks the same, or I can't find it. But [here's](http://www.school-for-champions.com/science/gravity_equations_falling_displacement.htm#.WdXiRROCzVo) another resource you can use to see the gravity displacement.
```

Comment:
The general equation is from kinematics, for displacement with constant acceleration. 
See http://www.physicsclassroom.com/class/1DKin/Lesson-6/Kinematic-Equations.

Where d is the displacement, v(i) is initial velocity, a is acceleration (in this case, a = g) and t is time